.travis.yml

```diff
@@ -6,6 +6,9 @@ cache:
   directories:
    - $HOME/.m2
 
+# skip install
```

Comment:
I know, hence the comment, but that's what Travis docs recommend: https://docs.travis-ci.com/user/customizing-the-build/#Skipping-the-Installation-Step